src/main/bash/sdkman-init.sh

```diff
@@ -148,3 +148,5 @@ if [[ "$sdkman_auto_env" == "true" ]]; then
 
 	sdkman_auto_env
 fi
+
+[[ "$bash_shell" == "true" ]] && source "${SDKMAN_DIR}/src/bash-completion.sh"
```

Comment:
How about we put this in the `bin` folder instead? Also, can we check that the file is present before we source it, otherwise this will break when we merge to master (we still need to find a reliable way of placing it in the archive and unpacking it onto all our users' systems in the right place).

Reply:
So, a dedicated file in the bin folder? Will this still be called from the init script?

Reply:
Yes, I'd place it in the bin folder next to the init script, making it easier for it to be sourced. It also means we don't need to do anything with weird relative paths. The archive is bundled together with Gradle, but then extracted using the `install.sh` and `selfupdate.sh` scripts in the [sdkman-hooks](https://github.com/sdkman/sdkman-hooks/tree/master/app/views) service.

Reply:
Here is where the init script is moved in `install.sh`: https://github.com/sdkman/sdkman-hooks/blob/master/app/views/install.scala.txt#L262
Here is the equivalent in selfupdate: https://github.com/sdkman/sdkman-hooks/blob/master/app/views/selfupdate.scala.txt#L101

Reply:
Hi @marc0der, one last question: The bash-completion.sh script should still be put into the `src` folder, right?